### DISCUSSION TITLE
Definition of energy in training and fine-tuning PES

### DISCUSSION THREAD

**Discussion by dengtq** [materialyzeai/matgl#204] [Q&A]
I'm recently trying M3GNet PES for dynamics and relaxation, which is powerful, and would like to explore more in fine-tuning. However, I have a simple (maybe stupid) question: for fine-tuning labels, should I use VASP e_0 directly (with the same MP parameters), or subtract the elemental reference energy similar to the original paper?

Look forward to your kind reply.

Best Regards,
Tianqi Deng
PI, HIC, Zhejiang University

**Comment by shyuep** [comment]
The Potential class already does the subtraction of elemental reference energies.
